client/systems.go

```diff
@@ -70,3 +73,24 @@ func (client *Client) ListSystems() ([]System, error) {
 	}
 	return rsp.Systems, nil
 }
+
+// RequestSystemAction issues a request to run the given seed system in the
+// provided mode.
+func (client *Client) RequestSystemAction(systemLabel, actionMode string) error {
```

Comment:
Maybe this should be something like:

DoSystemAction(sysAction *SystemAction) error ?

Reply:
As discussed on IRC we'll do `DoSystemAction(label, sysAction *SystemAction)` and POST to `/v2/systems/{label}`

Reply:
This is done now.